test/goose/factories.clj

```diff
@@ -38,14 +38,16 @@
     cron-opts))
 
 (defn create-dead-job [& [overrides]]
-  (let [error-state {:state {:error           "Error"
-                             :last-retried-at (u/epoch-time-ms),
-                             :first-failed-at 1701344365468,
-                             :retry-count     27,
+  (let [now (u/epoch-time-ms)
+        error-state {:state {:error           "Error"
+                             :last-retried-at now
```

Comment:
Missed out  adding `died-at` for a failed job. Check [here](https://github.com/nilenso/goose/blob/26e80d9b3f392cd9e08bb2790e503bc1f89d854e/src/goose/brokers/rmq/retry.clj#L33).